.github/workflows/prod-sync-data-prepper.yml

```diff
@@ -21,13 +21,14 @@ jobs:
     - name: S3 sync artifacts
       run: |
         #!/bin/bash
```

Comment:
We can remove this, as git actions assigns the bash as default.

Reply:
Removed.